Modules/StudyProgramme/classes/class.ilStudyProgrammeMailMemberSearchTableGUI.php

```diff
@@ -7,14 +7,14 @@ class ilStudyProgrammeMailMemberSearchTableGUI extends ilTable2GUI
     /**
      * @inheritdoc
      */
-    public function __construct(ilStudyProgrammeMailMemberSearchGUI $parent_obj, string $parent_cmd = "")
+    public function __construct(ilStudyProgrammeMailMemberSearchGUI $parent_obj, int $root_ref_id, string $parent_cmd = "")
     {
         global $DIC;
 
         $this->ctrl = $DIC['ilCtrl'];
         $this->lng = $DIC['lng'];
 
-        $obj_id = ilObject::_lookupObjectId($parent_obj->ref_id);
+        $obj_id = ilObject::_lookupObjectId($root_ref_id);
```

Comment:
Are we ever interested in the `$root_ref_id`? Why not directly ask for an `$root_obj_id` instead?

Reply:
done